### PR TITLE
git, git-abort, git-add, git-alias, git-cherry-pick, git-clone: add Korean translation

### PR DESCRIPTION
**Pull request by AMATEURTOSS**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

list of command
* git
* git abort
* git add
* git alias
* git cherry-pick
* git clone

### CODE REVIEW

File: pages.ko/common/git-add.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# git add
+
+> Adds changed files to the index.
+> 변경된 파일들을 인덱스에 추가합니다.
+> 더 많은 정보: <https://git-scm.com/docs/git-add>.
+
+- 인덱스에 파일 추가
```
Comment (blueskyson) [inline pages.ko/common/git-add.md:7]:
```suggestion
- 인덱스에 파일 추가:
```

File: pages.ko/common/git-add.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+`git add {{파일/의/경로}}`
+
+- 모든 파일 추가 (추적된 파일과 추적되지 않은 파일 모두):
+
+`git add -A`
+
+- 이미 추적된 파일만 추가
```
Comment (blueskyson) [inline pages.ko/common/git-add.md:15]:
```suggestion
- 이미 추적된 파일만 추가:
```

File: pages.ko/common/git-alias.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# git alias
+
+> Git 명령어 단축어 생성
```
Comment (blueskyson) [inline pages.ko/common/git-alias.md:3]:
```suggestion
> Git 명령어 단축어 생성.
```

File: pages.ko/common/git-add.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# git add
+
+> Adds changed files to the index.
```
[the remark this answers is not in the document]
Comment (AMATEURTOSS) [reply pages.ko/common/git-add.md:3]: I forgot to delete
The translation of line number 3 is line number 4.

**Review by blueskyson — APPROVED** [review]
Thanks for contribution.

**Pull request merged**